src/databricks/labs/remorph/coverage/remorph_snow_transpilation_coverage.py

```diff
@@ -6,8 +6,8 @@
 from databricks.labs.remorph.snow.snowflake import Snow
 
 if __name__ == "__main__":
-    input_dir = commons.get_env_var("INPUT_DIR", required=True)
-    output_dir = commons.get_env_var("OUTPUT_DIR", required=True)
+    input_dir = commons.get_env_var("INPUT_DIR", required=True) or ""
```

Comment:
This is wrong, we need to update the rule, to ignore this.

Reply:
Done